.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
is sudo a need? seems it significantly increases boot time of travis:
https://docs.travis-ci.com/user/ci-environment/
Also this page says "sudo: required" instead of "sudo: true", but maybe I looking at wrong page.
The point here is that using sudo seems to have effect, is it needed really?